patches/0032-DisableTagSchemaCheck-for-PF-provider.patch

```diff
@@ -67,6 +67,31 @@ index 0000000000..f790acb4e2
 +	r.Schema(ctx, resource.SchemaRequest{}, &resp)
 +	return &resp
 +}
+diff --git a/internal/provider/provider_tagcheck.go b/internal/provider/provider_tagcheck.go
+index 35202ebd58..28995620f2 100644
+--- a/internal/provider/provider_tagcheck.go
++++ b/internal/provider/provider_tagcheck.go
+@@ -18,9 +18,18 @@ func schemaMapForTagsChecking(ctx context.Context, r *schema.Resource, tagsCompu
+ 	switch flag := flag.(type) {
+ 	case bool:
+ 		if flag {
++			//lintignore:S013
+ 			return map[string]*schema.Schema{
+-				names.AttrTags:    &schema.Schema{Computed: tagsComputed},
```

Comment:
That file is created in this [patch](https://github.com/pulumi/pulumi-aws/blob/8771ec64099fe5c257fd343e0f9a976a5458a84a/patches/0030-Optimize-startup-performance.patch#L29). Imho we should correct it there, otherwise we'll make it harder to maintain that patch

Reply:
oh woops, I picked the wrong one! Good catch

Reply:
Updated!